app/scenes/Invite.js

```diff
@@ -1,213 +1,223 @@
-
-    return (
-      <form onSubmit={this.handleSubmit}>
-        {team.guestSignin ? (
-          <HelpText>
-            Invite team members or guests to join your knowledge base. Team
-            members can sign in with {team.signinMethods} or use their email
-            address.
-          </HelpText>
-        ) : (
-          <HelpText>
-            Invite team members to join your knowledge base. They will need to
-            sign in with {team.signinMethods}.{" "}
-            {can.update && (
-              <>
-                As an admin you can also{" "}
-                <Link to="/settings/security">enable email sign-in</Link>.
-              </>
-            )}
-          </HelpText>
-        )}
-        {team.subdomain && (
-          <CopyBlock>
-            <Flex align="flex-end">
-              <Input
-                type="text"
-                value={team.url}
-                label="Want a link to share directly with your team?"
-                readOnly
-                flex
-              />
-              &nbsp;&nbsp;
-              <CopyToClipboard text={team.url} onCopy={this.handleCopy}>
-                <Button
-                  type="button"
-                  icon={<LinkIcon />}
-                  style={{ marginBottom: "16px" }}
-                  neutral
-                >
-                  {this.linkCopied ? "Link copied" : "Copy link"}
-                </Button>
-              </CopyToClipboard>
-            </Flex>
-            <p>
-              <hr />
-            </p>
-          </CopyBlock>
-        )}
-        {this.invites.map((invite, index) => (
-          <Flex key={index}>
-            <Input
-              type="email"
-              name="email"
-              label="Email"
-              labelHidden={index !== 0}
-              onChange={(ev) => this.handleChange(ev, index)}
-              placeholder={`example@${predictedDomain}`}
-              value={invite.email}
-              required={index === 0}
-              autoFocus={index === 0}
-              flex
-            />
-            &nbsp;&nbsp;
+  }, [ui, t]);
+
+  return (
+    <form onSubmit={handleSubmit}>
+      {team.guestSignin ? (
+        <HelpText>
+          <Trans
+            defaults="Invite team members or guests to join your knowledge base. Team members can sign in with {{signinMethods}} or use their email
+          address."
+            values={{ signinMethods: team.signinMethods }}
+          />
+        </HelpText>
+      ) : (
+        <HelpText>
+          <Trans
+            defaults="Invite team members to join your knowledge base. They will need to
+          sign in with {{signinMethods}}."
+            values={{ signinMethods: team.signinMethods }}
+          />{" "}
+          {can.update && (
+            <Trans>
+              As an admin you can also{" "}
+              <Link to="/settings/security">enable email sign-in</Link>.
+            </Trans>
```

Comment:
Picked this from https://react.i18next.com/latest/trans-component#using-with-react-components

Reply:
👍 Yep, this is fine when there is no interpolation